python/lsst/ts/standardscripts/maintel/m1m3/check_actuators.py

```diff
@@ -0,0 +1,226 @@
+        )
+
+    @classmethod
+    def get_schema(self, cls):
+
+        m1m3_actuator_ids_str = ', '.join(str(fa) for fa in self.m1m3_actuator_ids)
+
+        url = "https://github.com/lsst-ts/"
+        path = (
+            "ts_externalscripts/blob/main/python/lsst/ts/standardscripts/"
+            "maintel/m1m3/check_actuators.py"
+        )
+        schema_yaml = f"""
+        $schema: http://json-schema.org/draft-07/schema#
+        $id: {url}{path}
+        title: CheckAcutators v1
+        description: Configuration for Maintel bump test SAL Script.
+        type: object
+        properties:
+            actuators:
+                description: Actuators to run the bump test.
+                oneOf:
+                  - type: array
+                    items:
+                      type: number
+                      enum: [{m1m3_actuator_ids_str}]
+                    minItems: 1
+                    uniqueItems: true
+                    additionalItems: false
+                  - type: string
+                    enum: ["all"]
+                default: "all"
+        additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+   
+    async def configure(self, config):
+        """Configure the script.
+        
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Configuration
+        """
+
+        # Actuators defined as "all" (ID) or a list of actuators (INDEX)
+        self.actuators = (
+            self.m1m3_actuator_ids if config.actuators == "all" else config.actuators
+        )
+
+        # Check if actuators are valid
+        if not all([actuator in self.m1m3_actuator_table_id for actuator in self.actuators]):
+            raise ValueError("Invalid actuator ID.")
+
+    def set_metadata(self, metadata):
+        
+        metadata.duration = self.timeout_one_bump * (self.total_actuators + self.total_sec_actuators) 
+
+    def is_actuator_dual(self, actuator_id: int) -> bool:
+        
+        # Converting actuator id to index
+        actuator_index = self.mtcs.get_m1m3_actuator_index(actuator_id)
+
+        is_dual = FATABLE[actuator_index][M1M3FATable.FATABLE_TYPE] == 'DAA'
+
+        return is_dual
+
+    async def run(self):
+
+        await self.assert_feasibility()
+        start_time = time.time()
+
+        # Get M1M3 detailed state
+        state = await self.mtcs.rem.mtm1m3.evt_detailedState.aget()
+        m1m3_detailed_state_name = self.MTM1M3.DetailedState(state.detailedState).name
+
+        self.log.info(f"M1M3 state before starting bump test: {m1m3_detailed_state_name}.")
+        
+        for i, actuator_id in enumarate(self.actuators): 
+            
+            await self.checkpoint(f"Running bump test on FA {actuator_id}.")
+          
+            # Determine if actuator has a secondary actuator
+            secondary_exist = self.is_actuator_dual(actuator_id)
+
+            if secondary_exist:
```

Comment:
So, instead of this, consider simply doing:

```
await self.mtcs.run_m1m3_actuator_bump_test(
    actuator_id=actuator_id, 
    primary=True, 
    secondary=secondary_exist,
)
```

Reply:
I think you need to remove this `if` statement. otherwise the test won't run if `secondary_exist` is `False`

Reply:
Yeah, I was just reviewing that. Thanks for catching this too!